### DISCUSSION TITLE
od-microsoft-edge-z-powrotem-do-google-chrome/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#113] [General]
# od-microsoft-edge-z-powrotem-do-google-chrome/

Kiedy zdałem sobie sprawę, że Microsoft Edge nie jest idealny dla wszystkich, zwłaszcza tych, którzy dużo inwestują w usługi Google, zmieniam swoje podejście i ponownie oceniam, gdzie Google Chrome jest lepszym rozwiązaniem.

https://dariusz.wieckiewicz.org/od-microsoft-edge-z-powrotem-do-google-chrome/

**Comment by xpil** [comment]
Używam Firefoxa od ponad 20 lat i jak na razie nie widzę powodów do przesiadki. Może z wyjątkiem niektórych środowisk korporacyjnych, gdzie integracja z resztą klamotów wymusza używanie Edge.

Tak że tak...
